supertokens_python/recipe/passwordless/api/implementation.py

```diff
@@ -72,13 +73,15 @@ async def create_code_post(self,
             elif isinstance(api_options.config.contact_config, (ContactEmailOrPhoneConfig, ContactPhoneOnlyConfig)):
                 if phone_number is None:
                     raise Exception("Should never come here")
-                await api_options.config.contact_config.create_and_send_custom_text_message(CreateAndSendCustomTextMessageParameters(
+                sms_input = TypePasswordlessSmsDeliveryInput(
                     phone_number=phone_number,
                     user_input_code=user_input_code,
                     url_with_link_code=magic_link,
                     code_life_time=response.code_life_time,
-                    pre_auth_session_id=response.pre_auth_session_id
-                ), user_context)
+                    pre_auth_session_id=response.pre_auth_session_id,
+                    user_context=user_context
```

Comment:
You need to be consistent - either pass user_context as a separate arg in all functions, or as part of the object in all functions.

Reply:
Fixed. Passing them seperately.